File: src/main/java/com/ebuozturk/ecommerce/dto/CartDto.java
package com.ebuozturk.ecommerce.dto;

import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.util.Date;

@Getter
@EqualsAndHashCode
@NoArgsConstructor
@AllArgsConstructor
public class CartDto {

    private Long id;
    private String name;
    private String no;
    private Date expiryDate;
    private String cvc;
}

File: src/main/java/com/ebuozturk/ecommerce/dto/cart/CartDto.kt
package com.ebuozturk.ecommerce.dto.cart;

import java.util.*


data class CartDto @JvmOverloads constructor(
    val id:Long,
    val name:String,
    val no:String,
    val expiryDate: Date,
    val cvc:String
){

}
